### PR TITLE
doc: Updates the helm doc with secret and storageclass configurations

### LINKED ISSUE #2255: Helm chart does not create secret and storageclass
# Describe the bug #

when setting in the values.yaml file
storageClass:
  create: true

and 

secret:
  create: true

these are not created on helm install --values as decribed in the README file here 
https://github.com/ceph/ceph-csi/tree/devel/charts/ceph-csi-rbd

# Environment details #

- Helm chart version : 3.3.1

### PR DESCRIPTION
**Pull request by yati1998**
This commit updates the helm doc with
storageclass and secret configurations.

Fixes: #2255

Signed-off-by: Yati Padia <ypadia@redhat.com>

### CODE REVIEW

**Review by nixpanic — CHANGES_REQUESTED** [review]
Can you please update the subject of the commit? The title should mention the added options a little instead of the uninformative "updates helm doc".

**Review by nixpanic — DISMISSED** [review]

File: charts/ceph-csi-cephfs/README.md [hunk 1]
```diff
@@ -77,62 +77,66 @@ kubectl delete namespace ceph-csi-cephfs
...
+| `driverName`                               | Name of the csi-driver                                                                                                                               | `cephfs.csi.ceph.com`                              |
+| `configMapName`                            | Name of the configmap which contains cluster configuration                                                                                           | `ceph-csi-config`                                  |
+| `externallyManagedConfigmap`               | Specifies the use of an externally provided configmap                                                                                                | `false`                                            |
+| `storageClass.create`                      | Specifies whether the Storage class should be created                                                                                                | `false`                                            |
```
Comment (Yuggupta27) [inline charts/ceph-csi-cephfs/README.md]:
```suggestion
| `storageClass.create`                      | Specifies whether the StorageClass should be created                                                                                                | `false`                                            |
```

File: charts/ceph-csi-rbd/README.md [hunk 2]
```diff
@@ -77,70 +77,74 @@ kubectl delete namespace ceph-csi-rbd
...
+| `configMapName`                            | Name of the configmap which contains cluster configuration                                                                                           | `ceph-csi-config`                                  |
+| `externallyManagedConfigmap`               | Specifies the use of an externally provided configmap                                                                                                | `false`                                            |
+| `kmsConfigMapName`                         | Name of the configmap used for encryption kms configuration                                                                                          | `ceph-csi-encryption-kms-config`                   |
+| `storageClass.create`                      | Specifies whether the Storage class should be created                                                                                                | `false`                                            |
```
Comment (Yuggupta27) [inline charts/ceph-csi-rbd/README.md]:
```suggestion
| `storageClass.create`                      | Specifies whether the StorageClass should be created                                                                                                | `false`                                            |
```

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]
To configure the deployment of StorageClass ad Secrets, apart from just `name` and `create`, there are multiple parameters that can be set by the user. Can you please add those too?
See [this](https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-rbd/values.yaml#L235) for reference

**Comment by yati1998** [comment]
> To configure the deployment of StorageClass ad Secrets, apart from just `name` and `create`, there are multiple parameters that can be set by the user. Can you please add those too?
> See [this](https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-rbd/values.yaml#L235) for reference

Sure

**Review by nixpanic — APPROVED** [review]

**Review by humblec — APPROVED** [review]

**Pull request merged**